source/agora/network/NetworkManager.d

```diff
@@ -1029,29 +1028,6 @@ public class NetworkManager
         this.validators().each!(v => v.client.sendBlockSignature(block_sig));
     }
 
-    /***************************************************************************
-
-        Sends the enrollment request to all the listeners.
-
-        Params:
-            enroll = the enrollment data to send
-
-    ***************************************************************************/
-
-    public void sendEnrollment (Enrollment enroll) @safe
-    {
-        foreach (ref node; this.peers[])
-        {
-            if (this.banman.isBanned(node.client.address))
-            {
-                log.trace("Not sending to {} as it's banned", node.client.address);
```

Comment:
Is it ok if the `BanMan` log is removed?

Reply:
> The "banned" messages were removed, as if a node is banned, it should not be in the list of peers to begin with.

https://github.com/bpfkorea/agora/pull/1576#issue-563969595